src/components/Link/RouterLink.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { ComponentProps } from './Link.types';
+import { Link as ReactRouterLink } from 'react-router-dom';
+
+import classNames from 'classnames';
+import styles from './Link.module.scss';
+
+export interface RouterLinkProps extends ComponentProps {
+  to: string;
+  replace?: boolean;
+}
+
+export const RouterLink = ({ to, replace, children, className }: RouterLinkProps) => {
+  console.log('got classname:', className);
```

Comment:
console.log?

Reply:
Thanks for picking it up - leaving it in for now while I work on link but will make sure to delete before submitting again.